### DISCUSSION TITLE
Need help unpausing my vercel account

### DISCUSSION THREAD

**Discussion by builtlikearatt** [vercel/community#6637] [Account Issue]
### Question

It's been 30 days and I'm trying to unpause my vercel account for my links

**Comment by amyegan** [comment]
Hey @Bryandrr. It looks like your Hobby account was paused **less than 30 days** ago because of usage exceeding your included [limits](https://vercel.com/docs/limits/overview). It appears to still be over the limit so it cannot be unblocked at the moment.

All plans receive [notifications](https://vercel.com/docs/dashboard-features/notifications#on-demand-usage-notifications) by email and on the dashboard when they are approaching and exceed their usage limits. You should have received a notification with more information about this usage and next steps.
